CMakeLists.txt

```diff
@@ -57,6 +62,23 @@ if(ASAGI)
     pkg_check_modules (ASAGI REQUIRED IMPORTED_TARGET asagi)
 endif()
 
+if (PYTHON_BINDINGS)
+    set(PYBIND11_VER 2.11.1)
```

Comment:
Please add `set(PYBIND11_FINDPYTHON ON)` for compability with new cmake versions, c.f. https://github.com/pybind/pybind11/blob/master/tools/FindPythonLibsNew.cmake#L102-L108